src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+// import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+ 
+public class Intake extends SubsystemBase{
+    private final DigitalInput input0 = new DigitalInput(0);
+    private final DigitalInput input1 = new DigitalInput(1);
+    private final DigitalInput input2 = new DigitalInput(2);
```

Comment:
This is pretty vague. If they are referring to the sensors then I would just make the name the location of the sensor. Otherwise just come up with a more descriptive name.